CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 # SAS Viya Monitoring for Kubernetes
 
 ## Unreleased
+
 * **Overall**
   * [FEATURE] Deploying into an air-gapped environment using local Helm archive (.tgz) files is now supported.  In this scenario,
 container images are still expected to be pulled from private container registry.  See
 [Configure SAS Viya Monitoring for Kubernetes for an Air-Gapped Environment](https://documentation.sas.com/?cdcId=obsrvcdc&cdcVersion=default&docsetId=obsrvdply&docsetTarget=n0grd8g2pkfglin12bzm3g1oik2p.htm).
   * [FIX] Ensure forced updates of Helm repos actually occur
 
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 45.28.0 to 48.3.2
+  * [UPGRADE] - Prometheus has been upgraded from version 2.44.0 to 2.45.0
+  * [UPGRADE] - Prometheus Operator has been upgraded from version 0.65.1 to 0.66.0
+  * [UPGRADE] - Grafana has been upgraded from version 9.5.5 to 10.0.3
+  * [UPGRADE] - Kube State Metrics has been upgraded from version 2.8.2 to 2.9.2
+  * [UPGRADE] - K8s-sidecar used with Grafana has been upgraded from version 1.24.0 to 1.24.6
+  * [UPGRADE] - Node Exporter has been upgraded from version 1.5.0 to 1.6.0
```

Comment:
I am seeing quay.io/prometheus/node-exporter:v1.6.1 being deployed in my environment. Please update CHANGELOG.md or code as appropriate.